capstone-backend/app.js

```diff
@@ -102,7 +117,11 @@ app.get('/profile/interests', async(req, res) => {
   const currentUser = Parse.User.current();
   if (currentUser){
     const userSkills = await getInterestQuery(currentUser, "Skills");
-    res.send({skills: userSkills})
+    const skillsData = fs.readFileSync('data/skills.json');
+    console.log('skillsData: ', skillsData);
+    const skillsJson = await JSON.parse(skillsData)
+    console.log('skillsJson: ', skillsJson);
+    res.send({skills: userSkills, skillsJson: skillsJson.skills, message: "got user interests ", typeStatus:"success" })
```

Comment:
Nit: I don't think skillsJson is a good name. I know you are making the distinction between user skills, and "all" available skills.
In any case, it's good to acknowledge that naming is hard, and it is also an valuable skill.